clang/test/CodeGenSYCL/function-attrs.cpp

```diff
@@ -29,21 +29,10 @@ int foo() {
 }
 
 template <typename Name, typename Func>
-__attribute__((sycl_kernel)) void kernel_single_task(const Func &kernelFunc) {
+[[clang::sycl_external]] void kernel_single_task(const Func &kernelFunc) {
```

Comment:
I think it was meant to be a kernel, so sycl_kernel_entry_point attribute would be appropriate here.
```suggestion
[[clang::sycl_kernel_entry_point]] void kernel_single_task(const Func &kernelFunc) {
```

Reply:
Agreed. It should have been [[clang::sycl_kernel_entry_point(Name)]]